CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 **NOTE:** With the open-sourcing of PromLens by [PromLabs](https://promlabs.com/), we decided to reset the latest proprietary version from 1.1.0 back to 0.1.0. We still list the proprietary version history in the changelog below for reference, denoted by a `promlabs-` version prefix.
 
+## 0.3.0 / 2022-12-02
+
+* [SECURITY] Update Exporter Toolkit (CVE-2022-46146) #66
```

Comment:
I just noticed that we don't add colons after `[...]` in Prometheus, but all the older changelogs in PromLens did. Maybe make those older entries consistent as well?